### DISCUSSION TITLE
Build Unikraft with grub support

### DISCUSSION THREAD

**Discussion by cristian-vijelie** [unikraft/unikraft#264] [Ideas]
It would be nice to add to the build system the option to automatically create an image compatible with grub (or other external bootloaders). This would aid those that want to use an external bootloader with Unikraft.

**Comment by razvand** [comment]
@cristian-vijelie, please open up [an issue](https://github.com/unikraft/unikraft/issues) for this, if not already out there.

**Comment by cristian-vijelie** [comment]
It was added some time ago
https://github.com/unikraft/unikraft/issues/265
